### PR TITLE
MaxSpeed isn't set all the time - Fixed Multiworld-Placing

### PR DESCRIPTION
**Pull request by Jakllp**
Fixed:
- Placing a Minecart when multiworld-support is turned off actually works now

Tweaked:
- MaxSpeed isn't set all the time now

Internal:
Bump to 1.1.1

### CODE REVIEW

File: src/main/java/com/useful/ucars/uCarsListener.java [hunk 1]
```diff
@@ -745,9 +745,8 @@ public void onUcarUpdate(ucarUpdateEvent event) {
 			}
 			UEntityMeta.removeMetadata(car, "car.jumping");
 		}
-		if(car instanceof Minecart) {
-			((Minecart)car).setMaxSpeed(5); // Don't allow game breaking speed - but faster
-			// than default
+		if(car instanceof Minecart && ((Minecart) car).getMaxSpeed() != 5) {
+			((Minecart)car).setMaxSpeed(5); // Don't allow game breaking speed - but faster than default
```
Comment (storm345) [inline src/main/java/com/useful/ucars/uCarsListener.java:749]: Was this having some performance implication? Not sure what prompted this change

File: src/main/java/com/useful/ucars/uCarsListener.java [hunk 2]
```diff
@@ -2190,4 +2189,8 @@ public void init() {
 	public List<String> getWorldList() {
 		return ucarworlds;
 	}
+	
+	public boolean getMultiverse() {
```
Comment (storm345) [inline src/main/java/com/useful/ucars/uCarsListener.java]: "isMultiverseEnabled" would be much clearer imho, or at the least "isMultiverse". (is convention for booleans)

**Review by storm345 — CHANGES_REQUESTED** [review]

File: src/main/java/com/useful/ucars/uCarsListener.java (hunk 1 above)
Comment (Jakllp) [reply src/main/java/com/useful/ucars/uCarsListener.java:749]: It seems to be slightly better for performance

Comment (storm345) [reply src/main/java/com/useful/ucars/uCarsListener.java:749]: Hmm I've never seen it show up in any timings

Comment (storm345) [reply src/main/java/com/useful/ucars/uCarsListener.java:749]: Yeah seems within spigot it is just directly writing to a single field, so should take basically no time whatsoever

Comment (Jakllp) [reply src/main/java/com/useful/ucars/uCarsListener.java:749]: I can change it back if you want

**Pull request merged**